modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -75,7 +76,7 @@
     protected boolean readonlyProperties = false;
     protected boolean swiftUseApiNamespace;
     protected String[] responseAs = new String[0];
-    protected String sourceFolder = "Classes" + File.separator + "OpenAPIs";
```

Comment:
Is this change related to some requirement of Vapor or SPM?

Reply:
Here I'm just following swift libraries' common conventions. https://swift.org/package-manager/#creating-a-library-package

Reply:
I like this change, and I think in the future we should adopt it, but this is a breaking change, so we can't merge it.

Reply:
I have been thinking about this and what we can do is to introduce a flag like `useSPMFileStructure` that is false by default to keep backwards compatibility.